grid/social-media/publish-post/maps/instagram.suma

```diff
@@ -108,3 +105,39 @@ operation MapApiError {
     original = body.error || body
   }
 }
+
+operation UploadMedia {
+  apiVersion = 'v14.0'
+  businessAccountId = args.businessAccountId
+  itemUrl = args.item ? args.item.url : undefined
+  isCarousel = args.isCarousel || undefined
+  children = args.children ? JSON.stringify(args.children) : undefined
```

Comment:
Should not this be just comma concatenate media ids? The example in Instagram docs do not contain json encoded array.

Reply:
I though so as well, but following [another sample request](https://developers.facebook.com/docs/instagram-api/guides/content-publishing#user-tags) in their docs, it seems they just expect JSON-encoded parameters. (and provably this works)

There's more weird stuff, like encouraging query parameters over JSON body payload. I think both your suggestion OR sending payload in the body would work, but I'd rather closely follow the docs.

Reply:
Thanks for explanation.